tests/test_accel.py

```diff
@@ -171,6 +173,8 @@ def test_angles():
 
 @timer
 def test_MG_3_4_accel():
+    # Exercise 3.4 from Montenbruck and Gill
```

Comment:
Please make this a docstring.

Reply:
@JasonBernstein1 Sorry, but did this one get missed? All of the others were turned into docstrings except this one.